.github/workflows/pr-main-infra-check.yml

```diff
@@ -1,7 +1,7 @@
-name: Main - Infra - Build and deploy
+name: Main - Infra Check - Build and deploy
```

Comment:
```suggestion
name: PR - Infra Check - Build and deploy
```

Reply:
But we are checking main's infra there, do you think we should make these changes?

Reply:
Looking at the other workflows, I think the naming scheme is {{ TRIGGER }} - {{ ACTION }}

Maybe a better name would be `PR - Test main infra`

Reply:
Yes it makes more sense.